### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by alongd**
Added issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+
+---
+
```
Comment (kspieks) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Is it worth prompting the user for installation information (OS, version for RMG-Py, RMG-database, ARC) like bug requests do for RMG-Py? Admittedly, I’ve rarely supplied this information, so this is up to you as it’s just a question.

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Question
+about: Ask a question about how to use T3
+title: ''
+labels: 'Type: Question'
+assignees: ''
+
+---
+
```
Comment (kspieks) [inline .github/ISSUE_TEMPLATE/question.md:9]: Can we add a phrase such as, “<-- Thanks for your interest in T3! Please first check the online documentation at <INSERT_URL_HERE> to see if it answers your question. Also, try searching GitHub issues to see if someone has asked your question before. --> 

For some reason, I get a 404 error when trying to load T3’s documentation at the moment so I can’t access the url. Does the page load for you?

Could we add similar short headers that encourage the user to search GitHub for the bug report and feature suggestion?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (alongd) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: I think we don't need to prompt, for the reason you specified (we can always ask if it's relevant and not supplied)

File: .github/ISSUE_TEMPLATE/question.md (hunk 2 above)
Comment (alongd) [reply .github/ISSUE_TEMPLATE/question.md:9]: Good idea! Added

**Pull request merged**